netmanager/src/views/pages/Download/Download.js

```diff
@@ -131,23 +208,18 @@ const Download = (props) => {
       outputFormat: outputFormat.value,
     };
 
-    // const dateDiff = moment(data.endDate).diff(moment(data.startDate), "days");
```

Comment:
thanks for removing this commented out code, good stuff! A good step towards cleaning up our codebase. @Codebmk